Library/.rubocop.yml

```diff
@@ -3,9 +3,13 @@ AllCops:
     - 'Homebrew/vendor/**/*'
     - 'Homebrew/test/vendor/**/*'
 
-# 1.8-style hash keys
+# Ruby 1.8 compatibility
```

Comment:
I don't have a strong preference. It felt nice to “reuse” this comment and to keep it at the top as a constant reminder for every reader of this file that we're still stuck with Ruby 1.8 (until this is no longer the case).